qvm.asd

```diff
@@ -92,4 +92,9 @@
                (:file "density-qvm")
                (:file "noisy-qvm")
                (:file "depolarizing-noise")
-               (:file "unitary-qvm")))
+               (:file "unitary-qvm")
+               (:module "error"
+                :serial t
+                :components ((:file "package")
```

Comment:
can you add a `README.md` to the `error/` directory with any general description and pertinent references? The README doesn't need to be exhaustively descriptive of how everything works.

(don't mention in the ASDF, just commenting here for relevance)